packages/components/src/data-store/api/user-vote.js

```diff
@@ -0,0 +1,34 @@
+import fetch from './fetch';
+
+export default {
+
+  fetch: async function({ projectId, type }) {
+    // todo, maar voor nu ff niet relevant
+    return {};
+  },
+
+  submitVote: async function({ projectId, type }, data) {
+
+    console.log('SUBMIT VOTE');
+
+    let url = `/api/project/${projectId}/vote`;
+    let headers = {
+      'Content-Type': 'application/json'
+    };
+
+    let votes = data.map(idea => ({
+      ideaId: idea.id,
+      opinion: 'selected',
+    }))
+
+    let json = await this.fetch(url, { headers, method: 'POST', body: JSON.stringify(votes)})
+
+    let event = new window.CustomEvent('osc-submit-user-vote', { detail: { type: 'userVote', votes } });
```

Comment:
Wat doen we op dit moment met deze events? Is dit niet iets wat alleen oud openstad alleen gebruikt? Gaat dit ook van toepassing zijn in headless?

Reply:
Ik denk dat je hooks wilt creeren, misschien voor gebruik tussen losse widgets op een pagina, maar ook voor externe toepassingen. Dat is zeker nog geen defiinitief uitgewerkt idee. Maar ook hier: laten we dit buiten deze PR houden.